lab_1_classify_by_unigrams/main.py

```diff
@@ -64,6 +120,20 @@ def detect_language(
     :return: a language
     """
 
+    if (not isinstance(unknown_profile, dict)
+            or not isinstance(profile_1, dict)
+            or not isinstance(profile_2, dict)):
+        return None
+    mse_1 = compare_profiles(unknown_profile, profile_1)
+    mse_2 = compare_profiles(unknown_profile, profile_2)
+    if isinstance(mse_1, float) and isinstance(mse_2, float):
+        if mse_1 < mse_2:
+            return str(profile_1['name'])
+        if mse_2 < mse_1:
+            return str(profile_2['name'])
+
+    return [profile_1['name'], profile_2['name']].sort()
```

Comment:
Looks like if `compare_profiles` return an incorrect value, this function will not return `None`

Reply:
Fixed